### DISCUSSION TITLE
i dont know how to add reference to the System.Printing.dll in .net 5 porject?

### DISCUSSION THREAD

**Discussion by sgf** [dotnet/csharplang#5214] [Q&A] [answered]
i dont know how to add reference to the System.Printing.dll in .net 5 porject,also cant get the package 
 by  search the nuget.

i have no idea for post this question to where.so im post here.
there's no framework assemblies on the reference manger.(in VS2019)

**Comment by YairHalberstadt** [comment]
This sounds like a good question for stack overflow. You can also try dotnet/runtime. This isn't the right platform as this is specifically for issues relating to the csharp language.

**Reply by yaakov-h** [reply]
That said, System.Printing looks like it's part of `net5.0-windows`.

**Comment by sgf** [comment, marked as answer]
this is the right Answer:
https://stackoverflow.com/questions/60398101/getting-the-access-to-the-system-printing-namespace-in-net-core-console-applica

Although you might doubt it.but thats the right way.

**Reply by yaakov-h** [reply]
That's only for .NET Core 3, the SDK structure changed in 5.

**Reply by sgf** [reply]
> That's only for .NET Core 3, the SDK structure changed in 5.

The same applies in .net 5. I just used this method to solve the compilation problem.
